File: src/main/org/nlogo/prim/etc/_nopatches.java
package org.nlogo.prim.etc ;

import org.nlogo.nvm.Context;
import org.nlogo.nvm.Reporter;
import org.nlogo.nvm.Syntax;

public final strictfp class _nopatches
	extends Reporter
{
	@Override
	public Object report( final Context context )
	{
		return world.noPatches() ;
	}
	@Override
	public Syntax syntax()
	{
		return Syntax.reporterSyntax( Syntax.TYPE_PATCHSET ) ;
	}
}

File: src/main/org/nlogo/prim/etc/_nopatches.scala
package org.nlogo.prim.etc

import org.nlogo.nvm.{ Context, Reporter, Syntax }

class _nopatches extends Reporter {
  override def syntax =
    Syntax.reporterSyntax(Syntax.TYPE_PATCHSET)
  override def report(context: Context) =
    world.noPatches
}
